ros2_ws/src/otto_deliberation/include/otto_deliberation/otto_autonomy.h

```diff
@@ -1,26 +1,23 @@
+#pragma once
 
-#include <chrono>
 #include <lock_service_msgs/srv/detail/lock__struct.hpp>
-#include <otto_deliberation/nav2_action_client.h>
-#include <otto_deliberation/robot_status.h>
-#include <otto_deliberation/tasks.h>
+#include "otto_deliberation/nav2_action_client.h"
+#include "otto_deliberation/robot_status.h"
```

Comment:
This was left on purpose - when someone will link to this project, somewhere, there will probably be whole include directory set, not necessary each single library. To me, with `"nav2_action_client.h"` there is a higher risk of having problems - having `#include "include/nav2_action_client.h"` instead of `#include "include/otto_deliberation/nav2_action_client.h"`.
I am obviously open to discussion - what is your opinion on this?